### PR TITLE
[DON'T MERGE] 1.0 support (based on #19)

### PR DESCRIPTION
**Pull request by filfat**
- [x] Update to protocol version _100_ (using 0.16 right now for testing)
- [ ] Handle Xbox connected players
- [ ] Things I've forgotten

### CODE REVIEW

**Comment by filfat** [comment]
see #19 for previous comments.

**Comment by mhsjlw** [comment]
@filfat can you post a screenshot of the chunk bug?

**Comment by mhsjlw** [comment]
@rom1504 should we merge this into another branch?

File: README.md [hunk 1]
```diff
@@ -26,3 +26,7 @@ This project is run by these guys:
   - [Filiph Sandström](https://github.com/filfat)
   - [mhsjlw](https://github.com/mhsjlw)
   - [rom1504](https://github.com/rom1504)
+
+## License
+Licensed under the MIT license.  
+ANY modifications/patches made by Filiph Sandström is licensed under the [ABRMS license](https://gist.github.com/filfat/d2ef7dffb50901064df06609d00f82ec).
```
Comment (ghost) [inline README.md]: Is this a joke?

Comment (filfat) [inline README.md]: No, but it's legal (https://github.com/filfat/pocket-minecraft-protocol/blob/1.0/LICENSE#L8 MIT allows for sub-licensing)

Comment (ghost) [reply README.md]: Ok, I'm going to have to close this PR then

Comment (filfat) [inline README.md]: I could of course retract the license for this project.

Comment (ghost) [reply README.md]: I appreciate it

Comment (roblabla) [reply README.md]: Although legal, don't expect a PR to be accepted if it modifies the project's license.

Imagine if every PR came with its special legal conditions. You'd get an unusable project in the end. Those lines are licensed MIT, those GPL, those APL, those ABRMS. Good luck figuring out which you're allowed to use :D (I totally want to create a project with broken licensing as a game to let people figure out how they can use it now).

Comment (ghost) [reply README.md]: Ha! I think we've figured it though, thankfully

**Pull request closed without merging**